torch/__init__.py

```diff
@@ -521,65 +521,130 @@ def is_warn_always_enabled():
 ################################################################################
 
 from ._tensor import Tensor
-from .storage import _StorageBase
+from .storage import _StorageBase, TypedStorage
 
-
-class DoubleStorage(_C.DoubleStorageBase, _StorageBase):
+class ByteStorage(_C.ByteStorageBase, _StorageBase):
     pass
 
```

Comment:
nit: probably want a note here explaining when it's appropriate to add new classes here (never) and what you should do instead

Reply:
Good idea, though what exactly should be done instead? Use `TypedStorage` directly, I suppose, but should we be any more specific than that?